Sources/MySQL/Protocol/MySQLCharacterSet.swift

```diff
@@ -19,25 +20,40 @@ struct MySQLCharacterSet: Equatable {
         self.raw = numericCast(byte)
     }
 
-    static var latin1_swedish_ci: MySQLCharacterSet = 0x0008
-    static var utf8_general_ci: MySQLCharacterSet = 0x0021
-    static var binary: MySQLCharacterSet = 0x003f
+    /// Creates a new `MySQLCharacterSet`
+    init(string: String) {
```

Comment:
Hmm, I think this should be a nil-fallable initializer in case the string is not recognized.

Reply:
I agree. But in this case would be better to make MySQLDatabaseConfig throws error.

```swift
public init(hostname: String = "127.0.0.1", port: Int = 3306, username: String, password: String? = nil, database: String, characterSet: String = "utf8_general_ci") throws {
  guard let charSet = MySQLCharacterSet(string: characterSet) else {
    throw MySQLError(identifier: "invalidCharacterSet", reason: "Cannot initialize \(MySQLCharacterSet.self) with value \(characterSet).", source: .capture())
  }

  self.hostname = hostname
  self.port = port
  self.username = username
  self.database = database
  self.password = password
  self.characterSet = charSet
}
```